run_node.sh

```diff
@@ -31,19 +38,38 @@ get_version () {
     # The version that is returned by the extrinsic looks like: 0.11.4-ae34eb4213
     # so we take the hash part and use it to identify docker images and commits in the repo.
 
-    if  [ -n "${MAINNET}" ]
+    if [ -n "${VERSION}" ]
     then
-        VERSION_RPC_URL="https://rpc.azero.dev"
-    else
-        VERSION_RPC_URL="https://rpc.test.azero.dev"
+        echo ""
+        echo -e "Version manually set to ${BGREEN}${VERSION}${NC}."
+        echo -e "Are you sure this is the correct version to run on ${BGREEN}${NETWORK}${NC}? [Y/n]"
```

Comment:
Comment for the future: we can pass any data function requires via its arguments, instead of using script globals. E.g. I was wondering here whether `NETWORK` is declared somewhere in this function, it's small to see that it is not, yet I needed to open IDE with whole file checked out from  branch to see that `NETWORK` is a global etc, and `VERSION` is not. Using function arguments we'll achieve better code encapsulation, so it will be less error prone.

Reply:
I will tackle this in a separate PR because it seems like it will require some thought.